src/common/timing/time-keeper.js

```diff
@@ -57,28 +55,23 @@ export class TimeKeeper {
    * @param rumRequest {XMLHttpRequest} The xhr for the rum request
    * @param startTime {number} The start time of the RUM request
    * @param endTime {number} The end time of the RUM request
+   * @param nrServerTime {number} the unix number value of the NR server time in MS, returned in the RUM request body
    */
-  processRumRequest (rumRequest, startTime, endTime) {
+  processRumRequest (rumRequest, startTime, endTime, nrServerTime) {
     this.processStoredDiff() // Check session entity for stored time diff
     if (this.#ready) return // Server time calculated from session entity
 
-    const responseDateHeader = rumRequest.getResponseHeader('Date')
-    if (!responseDateHeader) {
-      throw new Error('Missing date header on rum response.')
-    }
-    if (!rfc2616Regex.test(responseDateHeader)) {
-      throw new Error('Date header invalid format.')
-    }
+    if (!nrServerTime) throw new Error('nrServerTime not found')
```

Comment:
Should we have a default if BCS stops working aka giving nrservertime? So as not to "break" all new session pages on customers browser monitoring traffic.

Reply:
All data therein would be invalid since the timestamps would be wrong so the accepted behavior up to this point is to kill the agent if a time correction cant be made.  That becomes a DEM problem if BCS does not work as intended